mathics/builtin/physchemdata.py

```diff
@@ -10,10 +14,11 @@
 from mathics.core.expression import (Expression, from_python, Symbol, String,
                                      strip_context)
 from mathics.settings import ROOT_DIR
+import six
 
 
 def load_element_data():
-    element_file = open(ROOT_DIR + 'data/element.csv', 'rb')
+    element_file = open(ROOT_DIR + 'data/element.csv', 'r')
     reader = csvreader(element_file, delimiter='\t')
```

Comment:
The delimiter can't be unicode for Py2.

If there's a way to get a `str` (not unicode) literal with `six` we can import `unicode_literals` in this module too.